404.dd

```diff
@@ -1,12 +1,20 @@
 Ddoc
 
-$(D_S 404 Not Found,
+<div class="center-children">
```

Comment:
Replace HTML tags with macros like `$(DIV ..`.

Reply:
Ah this is page won't be part of the spec which is rendered to LaTeX, so it's fine to use plain old HTML here.